pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSchemaRestletResource.java

```diff
@@ -339,6 +363,7 @@ public String validateSchema(String schemaJsonString, @Context HttpHeaders httpH
   @GET
   @Produces(MediaType.APPLICATION_JSON)
   @Path("/schemas/fieldSpec")
+  @Authorize(targetType = TargetType.CLUSTER, action = Actions.Cluster.GET_FIELD_SPEC)
```

Comment:
This API is kind of special because it does not access any data in the cluster. Is there a way to always allow access?

Reply:
Yes, annotated with ManualAuthorization to skip from request filter, and no validation in the implementation.